testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-oauth2-https-proxy-authn/.htpasswd

```diff
@@ -0,0 +1 @@
+proxy_user:$apr1$M4Whksmo$csx/0NiGcPVE06m3AyQfz.
```

Comment:
please move files not related to trino to `proxy` directory:
`singlenode-oauth2-https-proxy-authn/proxy/.htpasswd`